### PR TITLE
add `nodePort: null` to traefik service ports config

### PR DESCRIPTION
**Pull request by shaunc**
Addresses deployment with service type `ClusterIP` -- https://github.com/dask/dask-gateway/issues/304

### CODE REVIEW

File: resources/helm/dask-gateway/values.yaml [hunk 1]
```diff
@@ -205,10 +205,12 @@ traefik:
       web:
         # The port HTTP(s) requests will be served on
         port: 80
+        nodePort: null
```
Comment (jcrist) [inline resources/helm/dask-gateway/values.yaml:208]: Odd, wouldn't the default be `nodePort: null` if it's not present? I'm confused why the existing check in the template results in an invalid spec. Will try to give this a closer look tomorrow morning.

Comment (shaunc) [reply resources/helm/dask-gateway/values.yaml:208]: Yes -- odd. Perhaps the way the templating system works, missing actually evaluates to an empty string, which then doesn't evaluate to "false". Still - presumably if its referred to as if its in "values.yaml", its good to actually have it there. The other option would be to take it out of "values.yaml" and also out of the service template.

Comment (shaunc) [reply resources/helm/dask-gateway/values.yaml:208]: (keep in mind that this is golang-based, so that python intuitions might not apply. :))

**Comment by jcrist** [comment]
Apologies for the delay here. I'm unable to replicate your issue locally (setting `traefik.service.type` to `ClusterIP` works fine for me). Not sure why you're having issues (perhaps a bug in an older version of helm?). The changes here aren't harmful though, happy to merge them. Thanks!

**Pull request merged**